app/_data/docs_nav_mesh_2.3.x.yml

```diff
@@ -1,680 +1,186 @@
-product: mesh
-release: 2.3.x
 generate: true
 assume_generated: true
-items:
-  - title: Introduction
-    icon: /assets/images/icons/documentation/icn-flag.svg
-    items:
-      - text: About service meshes
-        url: /introduction/what-is-a-service-mesh/
```

Comment:
the url for this one changed to `/introduction/about-service-meshes/`

Reply:
Yes to reflect Kuma: https://kuma.io/docs/2.2.x/introduction/about-service-meshes/